### PR TITLE
Adição das loss functions

### LINKED ISSUE #7: william: implementar funções de perda de classificação
implementar as seguintes funções de perda para classifificação
- Log-Likelihood Loss
- Binary Cross-Entropy Loss
- Softmax
- cross entropy loss
Image

Image

Image

Image

### PR DESCRIPTION
**Pull request by WilliamsASF**
revisa por favor

### CODE REVIEW

File: src/core/functions/LossFunctions.py
Comment (gabriel-ferreira-da-silva) [inline src/core/functions/LossFunctions.py:1]: tá top 🤙

**Review by gabriel-ferreira-da-silva — APPROVED** [review]
